src/main/java/dinkplugin/notifiers/DeathNotifier.java

```diff
@@ -276,4 +277,17 @@ private static List<SerializedItemStack> getStacks(ItemManager itemManager, List
             .collect(Collectors.toList());
     }
 
+    /**
+     * @param client {@link Client}
+     * @return whether the player is not in a safe area (excluding inferno and fight cave)
+     */
+    private static boolean isDangerous(Client client) {
+        if (!WorldUtils.isSafeArea(client))
+            return true; // normally dangerous
+
+        // inferno and fight cave are technically safe, but we want death notification regardless
+        int regionId = client.getLocalPlayer().getWorldLocation().getRegionID();
+        return WorldUtils.isInferno(regionId) || WorldUtils.isTzHaarFightCave(regionId);
```

Comment:
This is consistent with maintaining current behavior - in a future version we could make these config toggles if desired